crates/virtio-bindings/src/virtio_config.rs

```diff
@@ -0,0 +1,127 @@
+/* automatically generated by rust-bindgen 0.59.1 */
```

Comment:
The PR description says the bindings are generated with 0.6.0, but the comment says another thing. Also, shouldn't all the bindings in the repository be generated with the same version?

Reply:
Description is talking about Linux kernel version v6.0.

https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tag/?h=v6.0

And yes, maybe all bindings should be generated together.

Reply:
Ah never mind, I confused the kernel version with the rust-bindgen one, but the question still stands, are all the bindings generated with the same rust-bindgen version?

Reply:
@lauralt I have updated all bindigns to bindgen 0.60.1 and kernel v6.0 now.